harness/determined/cli/model.py

```diff
@@ -108,6 +108,11 @@ def list_versions(args: Namespace) -> None:
         _render_model_versions(model.list_versions())
 
 
+def delete(args: Namespace) -> None:
+    model = client.Determined(args.master, None).get_model(args.name)
```

Comment:
I would say to keep consistent with other commands, maybe do
```
 sess = cli.setup_session(args)
 model = model_by_name(sess, args.name)
```
to get the model

Reply:
Good call! Thanks